app/src/main/java/com/nextcloud/talk/utils/ApiUtils.java

```diff
@@ -127,7 +129,11 @@ public static String getRoom(String baseUrl, String token) {
     }
 
     public static String getRoomV3(String baseUrl, String token) {
-        return baseUrl + ocsApiVersion + "/apps/spreed/api/v3" + "/room/" + token;
+        return baseUrl + ocsApiVersion + spreedApiVersion3 + "/room/" + token;
```

Comment:
`"room"` should likely be extracted into a constant (private static final)

Reply:
So as the v-version is bound to the capability and specific to the controller, I wondered i we should use conversationApiVersion4, etc. which then is `/apps/spreed/api/v4/room/`, because e.g. chat is still on v1, settings is v2, conversation and participants v4, etc.

Reply:
That would also be fine too imho. In general it is just that unique strings used in several places should be made a constant to prevent typos etc. in one/some place. Declaring larger parts of the url for the different endpoints would be fine too.